test/unit/specs/store/pool.spec.js

```diff
@@ -1,64 +1,83 @@
-import setup from "../../helpers/vuex-setup"
-import lcdClientMock from "renderer/connectors/lcdClientMock.js"
 import poolModule from "renderer/vuex/modules/pool.js"
 
-const instance = setup()
-
-const mockPool = lcdClientMock.state.pool
-const mockRootState = {
-  connection: {
-    connected: true
-  }
-}
-
 describe(`Module: Pool`, () => {
-  let store
+  let module, state, actions, mutations
```

Comment:
We should avoid use reserved js words `module`

Reply:
agree, let's change it